### PR TITLE
added placeholder option for course year select

### PR DESCRIPTION
**Pull request by djvoa12**
**Summary:**
New course creation year dropdown has a placeholder.

**UI:**
Screen Shot 2019-06-03 at 12 53 55 PM

Screen Shot 2019-06-03 at 12 54 03 PM

Fixes https://github.com/ilios/common/issues/697

### CODE REVIEW

File: translations/en-us.yaml [hunk 1]
```diff
@@ -261,6 +261,7 @@ general:
   permissionsInOtherSchools: "Permissions in Other Schools"
   perPage: "Per Page"
   phone: Phone
+  pleaseSelect: "Please Select"
```
Comment (jrjohnson) [inline translations/en-us.yaml]: I like the generic, but everywhere else we have `Select School` or `Select Course` I think this one should be `Select Year`.

File: app/templates/components/new-course.hbs [hunk 2]
```diff
@@ -23,6 +23,9 @@
     <label>{{t "general.academicYear"}}:</label>
 
     <select data-test-year onchange={{action "setYear" value="target.value"}}>
+      <option disabled hidden selected value="">
```
Comment (jrjohnson) [inline app/templates/components/new-course.hbs]: My reading of the `hidden` attribute says this is invalid: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden thoughts?

It's also invalid to have more than one element selected this causes issues with keyboard navigation it should probably be `selected={{is-empty this.selectedYear}}` though I'm not sure about `is-empty`.

File: app/components/new-course.js [hunk 3]
```diff
@@ -34,22 +34,12 @@ export default Component.extend(Validations, ValidationErrorDisplay, {
   currentYear: null,
   selectedYear: null,
   title: null,
-  years: null,
   cancel() {},
   save() {},
+  years: Object.freeze(YEARS),
```
Comment (jrjohnson) [inline app/components/new-course.js]: ❤️ this. So much better than doing it in a life cycle hook.

File: translations/en-us.yaml (hunk 1 above)
Comment (djvoa12) [reply translations/en-us.yaml]: Fixed.

File: app/templates/components/new-course.hbs (hunk 2 above)
Comment (djvoa12) [reply app/templates/components/new-course.hbs]: Fixed.

**Comment by dartajax** [comment]
Functionally, I think this is fine. Is "Select Educational Year" correct as the caption? Should it be "Select Academic Year" instead? Question for @saschaben

**Comment by stopfstedt** [comment]
> Caption should be "Select Academic Year" rather than "Select Educational Year".

fixed.

**Comment by dartajax** [comment]
Save New Course is failing on demo3 right now as well so it may have nothing to do with this PR. If that is the case, my review can be labelled as "stale" once the save event is working again.

**Pull request merged**